tv_apps/tv_shared/components/smart-button/style.css

```diff
@@ -403,6 +406,32 @@ smart-button[type="circle-text"].pressed {
   background-color: #00caf2;
 }
 
+smart-button[type="circle-text"].primary {
+  background-color: #00caf2;
```

Comment:
Check with visual that if we need to have the transition of background-color and color.